### DISCUSSION TITLE
사랑의 기술

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#98] [General]
# 사랑의 기술

사랑은 다른 사람에게 적극적으로 침투하는 것이고, 이러한 침투를 통해 알려고 하는 나의 욕망은 합일에 의해 만족을 얻는다. 낮과 밤은 겉으로는 적이지만 동일한 목적에 이바지 하고 있고, 서로의 일을 완성하기 위해 밤과 낮은 서로 사랑하고 있다. 중요한 것은 이러한 일을 알아차리고, 이러한 일을 합리화하는 무수한 방법이 있더라도 결코 합리화하지 않는 것이다…

https://vero.wiki/book-love-skill/

**Comment by carsago** [comment]
읽는거 포기했다가 오늘 다시 읽고 있습니다.
사랑이 뭘까요 기술을 말하는거보몈 여기선 감정은 아닐텐데

**Reply by Cyma-s** [reply]
책 중에 이렇게 묘사 되어 있기는 하네요.

> 사랑은 사랑하고 있는 자의 생명과 성장에 대한 우리의 적극적 관심이다.

> 사랑은 인간에게 능동적인 힘이다. 곧 인간을 동료에게서 분리하는 벽을 허물어버리는 힘, 인간을 타인과 결합하는 힘이다. 

제가 느끼기에 사랑은 지속적인 관심이자, 상대를 그대로 받아들이고자 하는 힘, '생산적'인 활동을 하는 것의 원동력이라고 생각합니다.
결국 목적은 나를 발견하고, 내 삶을 가치 있는 방향으로 이끌어가고, 다른 사람을 이해하기 위함 아닐까 싶네요.
사람들을 사랑하며 나를 사랑하게 되고, 나를 사랑함으로써 곧 다시 사람들을 사랑할 수 있고, 사랑하는 나의 삶을 사랑하자는 함의가 멋진 것 같습니다.
